package.json

```diff
@@ -37,11 +37,14 @@
         "build": "hereby build",
         "build:watch": "hereby build:watch",
         "test": "hereby test",
+        "api:build": "npm run -w @typescript/api build",
         "extension:build": "npm run -w typescript-lsp build",
-        "extension:watch": "npm run -w typescript-lsp watch"
+        "extension:watch": "npm run -w typescript-lsp watch",
+        "node": "node --no-warnings --conditions @typescript/source"
     },
     "workspaces": [
-        "./_extension"
+        "./_extension",
```

Comment:
Maybe not in this PR, but I do wonder if we should just move the extension over there or something.

Reply:
That would be fine, but I can also see `_packages` as only stuff that’s going to get published to npm.

Reply:
Yeah, I think that's reasonable.